NuGet.Docs/Release-Notes/NuGet-2.8.7.md

```diff
@@ -1,6 +1,6 @@
 # NuGet 2.8.7 Release Notes
 
-[NuGet 2.8.5 Release Notes](nuget-2.8.5) | [NuGet 3.0 Preview Release Notes](nuget-3.0-preview)
+[NuGet 2.8.6 Release Notes](nuget-2.8.6) | [NuGet 2.9-RC Release Notes](nuget-2.8.8-RC)
```

Comment:
yes, previous versions of the release notes skipped versions in the header and this corrects them